infra/partner-vm-testnet/terraform-testnet-example.tfvars

```diff
@@ -3,11 +3,11 @@ env          = "testnet"
 node_configs = [
   {
     # Each node has a unique account ID
-    account              = "multichain-node-testnet-7.testnet"
+    account              = "{your_near_account_id}"
     cipher_pk            = "<your_cipher_pk>"
     # These 3 values below should match your secret names in google secrets manager
-    account_sk_secret_id = "multichain-account-sk-testnet-0"
-    cipher_sk_secret_id  = "multichain-cipher-sk-testnet-0"
-    sk_share_secret_id   = "multichain-sk-share-testnet-0"
+    account_sk_secret_id = "multichain-account-sk-testnet-{your_entity_name}"
```

Comment:
Each partner will have their own GCP account and Project.  As a result - a separate namespace for these parameters. Why do we need to include partner names here and how it will ease their lives?

Reply:
yeah we can use the same names